### ISSUE TITLE
AoT compilation kendo-angular-inputs error

### ISSUE THREAD

**Issue by SeriousLemon** [telerik/kendo-angular#442]
I get the following output when running AoT compilation:

```
ERROR in ng:///.../node_modules/@progress/kendo-angular-inputs/dist/es/maskedtextbox/maskedtextbox.component.d.ts.MaskedTextBoxComponent.html (13,13): Supplied parameters do not match any signature of call target.
ng:///.../node_modules/@progress/kendo-angular-inputs/dist/es/maskedtextbox/maskedtextbox.component.d.ts.MaskedTextBoxComponent.html (14,13): Supplied parameters do not match any signature of call target.
```

I'm not actually using the inputs, they are only added as a requisite for the grid.

**Comment by rusev** [comment]
Our AoT builds seems to work properly at the moment. We will look into it and will post back with more details.

**Comment by kurtvs** [comment]
We are also having this issue.

**Comment by Mart112358** [comment]
Same here with v0.21.4. We are using @angular/cli v1.0.0 (Angular 4). I didn't test with angular 2. It might be related to how AoT build changed with Angular 4, see [Angular 4.0.0 Now Available](http://angularjs.blogspot.ca/2017/03/angular-400-now-available.html)

**Comment by ajaysingh77** [comment]
I am also getting the same error with AOT and without compilation. 
ng build --prod --aot or ng --prod both commands causing the above issue. 
if I am using only ng build then things are working fine. Things was working fine before upgrade to Angular4. The current package versions than I am using are
 "@angular/cli": "1.0.0",
"@progress/kendo-angular-inputs": "^0.22.0"

**Comment by tsvetomir** [comment]
Should be fixed in `@progress/kendo-angular-inputs@0.22.1`

**Comment by kurtvs** [comment]
This fixed the issue indeed. Thanks!
